source/system/string/string_literal.c

```diff
@@ -21,20 +21,25 @@ static void mutex_free(mutex_t* m) { CloseHandle(*m); }
 #else
 
 typedef pthread_mutex_t mutex_t;
-#define MUTEX_INIT PTHREAD_MUTEX_INITIALIZER
 static void mutex_lock(mutex_t* m) { pthread_mutex_lock(m); }
 static void mutex_unlock(mutex_t* m) { pthread_mutex_unlock(m); }
 static void mutex_free(mutex_t*  m) { pthread_mutex_destroy(m); }
+static void mutex_init(mutex_t* m) { pthread_mutex_init(m, NULL); }
```

Comment:
I know windows is not our major target platform, but can you add this function for the windows build too ? Just for the sake of completeness